packages/web-components/fast-components/src/tooltip/tooltip.stories.ts

```diff
@@ -0,0 +1,39 @@
+import { STORY_RENDERED } from "@storybook/core-events";
+import addons from "@storybook/addons";
+import { FASTDesignSystemProvider } from "../design-system-provider";
+import TooltipTemplate from "./fixtures/base.html";
+import { FASTTooltip } from "./";
+
+// Prevent tree-shaking
```

Comment:
I'd prefer to see an explanation of why we are preventing tree-shaking in the comment.

Reply:
I think this comment is clear enough for the stories page. We prevent tree shaking here because without the Class defined or used in this it will be tree shaken out.